Mathlib/Topology/Order/ScottTopology.lean

```diff
@@ -142,7 +142,7 @@ def scottHausdorff (α : Type*) [Preorder α] : TopologicalSpace α where
     obtain ⟨b, hbd, hbds₀⟩ := h s₀ hs₀s hd₁ hd₂ hd₃ has₀
     exact ⟨b, hbd, Set.subset_sUnion_of_subset s s₀ hbds₀ hs₀s⟩
 
-variable (α) [TopologicalSpace α]
+variable (α) [t : TopologicalSpace α]
```

Comment:
You never use the name explicitly
```suggestion
variable (α) [TopologicalSpace α]
```

Reply:
It now fails with:
```
error: ././././Mathlib/Topology/Order/ScottTopology.lean:379:51: invalid argument name 't' for function 'Topology.IsScottHausdorff.isOpen_of_isLowerSet'
error: Lean exited with code 1
```

Reply:
Oh okay, maybe that's a sign it should rather be `{t : TopologicalSpace α}`

Reply:
Okay, how's that?

Reply:
Looks good, yes!